app/models/form_profiles/vha_10_10d.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class FormProfiles::VHA1010d< FormProfile
+  FORM_ID = '10-10D'
+
+  def metadata
+    {
+      version: 0,
+      prefill: true,
+      returnUrl: '/signer-type'
+    }
+  end
+end
```

Comment:
We did not previously have a form profile for 10-10d, so the returnUrl wasn't set. This prevented using the "Continue your application" button on user profile from redirecting correctly.